bin/compute-edge-version

```diff
@@ -11,7 +11,7 @@ bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 . "$bindir"/_tag.sh
 tag=$(named_tag)
 
-edge_tag_regex='edge-([0-9][0-9])\.([0-9]|[0-9][0-9])\.([0-9]+)'
+edge_tag_regex='edge-([0-9][0-9]).([0-9]|[0-9][0-9]).([0-9]+)'
```

Comment:
dots don't need to be escaped in this context. Doing so makes awk show a warning.